src/main/java/xyz/deseteral/lebkuchenfm/services/commands/CommandProcessor.java

```diff
@@ -0,0 +1,26 @@
+package xyz.deseteral.lebkuchenfm.services.commands;
+
+import xyz.deseteral.lebkuchenfm.domain.Command;
+import xyz.deseteral.lebkuchenfm.domain.CommandProcessingResponse;
+
+import java.util.List;
+import java.util.Optional;
+
+interface CommandProcessor {
+    CommandProcessingResponse process(List<String> args);
+
+    String getKey();
+
+    Optional<String> getShortKey();
+
+    String getHelpMessage();
+
+    default boolean matches(Command command) {
```

Comment:
Przeniósłbym to do osobnej klasy.

Reply:
Przeniosłem do klasy `Command`, wydaje mi się że to dobre miejsce